aten/src/ATen/native/native_functions.yaml

```diff
@@ -8792,6 +8792,14 @@
   python_module: linalg
   variants: function
 
+- func: linalg_svdvals(Tensor input) -> Tensor
+  python_module: linalg
+  variants: function
+
+- func: linalg_svdvals.out(Tensor input, *, Tensor(a!) out) -> Tensor(a!)
+  python_module: linalg
+  variants: function
```

Comment:
Please consider using structured kernels here! https://github.com/pytorch/rfcs/blob/rfc-0005/RFC-0005-structured-kernel-definitions.md

Reply:
@ezyang Are we ready to start requiring new ops use structured kernels?

Reply:
I wouldn't require it, but people should at least try (and report if there is something that make it hard to do)